packages/donor/src/screens/bookDonation/BookDonationScreen.stories.tsx

```diff
@@ -116,3 +121,17 @@ Fetching.args = {
   ...props,
   isFetching: true,
 };
+
+export const SlotTooCloseDoDonorLastAppointmentPopup = (
+  args: BookDonationScreenProps
+) => <BookDonationScreen {...args} />;
```

Comment:
This should be
`export const SlotTooCloseToDonorLastAppointmentPopup = Template.bind({});`

1. Please fix this.
2. Please adjust the other stories as well.
3. Notice the typo in the story name (do => to)

Reply:
![image](https://user-images.githubusercontent.com/30054656/163686566-a1b689c3-38d1-409f-9e0b-b502b3566281.png)
Cool to know ! Thank you for commenting this I learned something new

Reply:
Getting this error :  
```
 ● Storyshots › Screens/Book Donation Screen › Slot Too Close To Donor Last Appointment Popup

    TypeError: parentInstance.children.indexOf is not a function
```

When using the bind, have you encountered this before ? @yaron1m

Reply:
Can you share the entire file after your change?

Reply:
ok ok found the issue, it is because the pop up needs a DOM to render itself but in storybook snapshots there is no DOM. We don't have any other snapshots for this kind of popups. This popups are business related and need to be tested manually or with Testim.IO WDYT ?

Reply:
I understand. In this case let's just skip the story for this case